### ISSUE TITLE
No cooldowns for Business Class travel

### ISSUE THREAD

**Issue by auboli77** [Mephiles/torntools_extension#707]
I have found that when going to travel business class as a default, Cooldowns will not show up. That part is just missing. Workaround is to change way of travel to some other option like airstrip and then go back to business class travel and cooldowns are shown.

Using TornTools version 6.9.2 on Firefox 106.0.3 (64-bit).

**Comment by DeKleineKobini** [comment]
I'm afraid that this isn't something that would be easily fixed, purely because it requires someone in the correct company to check and fix it.

**Comment by Kwack-Kwack** [comment]
Would it be possible to fix with a BCT? I'm happy to send you the one I've got in my inv to let you test it, although from memory it might not change to business class by default. 

Currently bound to mobile so I can't even test this

**Comment by DeKleineKobini** [comment]
I got a BCT myself due to Halloween. But I also got some nice donation from someone which I can use to fund a lot of stuff for TT. It doesn't change the default though, so that does not work.

**Comment by auboli77** [comment]
I used inspection on the page as I opened travel section(at this point if I click on a destination I don't get the cooldowns). Second pastebin is same page but I clicked on standard travel type and then back to business class travel(at this point if I click on a destination I get the cooldowns). There are a few small differences. I only pasted the part I though was relevant. If I can do more to help let me know.
Before: [https://pastebin.com/PNa60Ca8](https://pastebin.com/PNa60Ca8)
After: [https://pastebin.com/JZ3hbDZz](https://pastebin.com/JZ3hbDZz)
